src/D2L.Bmx/SourceGenerationContext.cs

```diff
@@ -1,7 +1,7 @@
 using System.Text.Json.Serialization;
 using D2L.Bmx.Okta.Models;
 
-namespace D2L.Bmx.Okta;
+namespace D2L.Bmx;
```

Comment:
we have an editorconfig/analyzer rule to enforce namespace matching folder structure, but for some reason it's ineffective when there's a partial class inside.